pkg/tecdsa/dkg/preparams_storage.go

```diff
@@ -0,0 +1,158 @@
+package dkg
```

Comment:
I think we can put this code directly to `preparams.go` file because:
- `pkg/tecdsa/dkg` contains multiple files and this code is strictly connected with pre-params so it would be good to have it close to each other
- `preparams.go` is very small currently so this doesn't harm readability

Reply:
efd70e811556099f3fb52da0a546bb55ec727b3e